worldengine/simulations/wind.py

```diff
@@ -0,0 +1,102 @@
+import numpy
+from noise import snoise2
+
+class WindSimulation(object):
+
+    @staticmethod
+    def is_applicable(world):
+        return not world.has_wind()
+
+    def execute(self, world, seed):
+        assert seed is not None
+        direction = self._wind_direction(world, 0.5, seed)
+        world.set_wind_direction(direction)
+
+    @staticmethod
+    def _wind_direction(world, distorsion_factor, seed):
```

Comment:
I think this should be called `_calculate()` - that's how it is done in all the other modules, right?